src/style.css

```diff
@@ -0,0 +1,50 @@
+body {
+  margin: 0;
+}
+
+.container * {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+}
+
+.stars > div:last-child {
+  margin-right: 0;
+}
+
+.stars > div:first-child {
+  margin-left: 0;
+}
+
+.stars--5 {
+  margin-bottom: 0;
+}
```

Comment:
```suggestion
```
why do you need this styles?

Reply:
Without these styles the tests don't pass-)